src/scripts/Components/Hub.js

```diff
@@ -132,20 +132,24 @@ class Hub extends React.Component {
                 onInstall={this.handleUpdate}
                 onReload={this.handleReload}/>
               {
-                this.props.enableContentHub &&
+                this.props.enableContentHubSearch &&
                 <ReuseContent id="h5p-hub-reuse"
                               title={Dictionary.get('reuseContentTabLabel')}
                               isVisible={this.state.expanded && this.state.section === 'h5p-hub-reuse'}
                               getAjaxUrl={this.props.getAjaxUrl}
                               onDownload={this.handleUpload}
                 />
               }
-              <UploadContent id="h5p-hub-upload"
-                title={Dictionary.get('uploadTabLabel')}
-                getAjaxUrl={this.props.getAjaxUrl}
-                contentId={this.props.contentId}
-                setFocus={this.state.expanded && this.state.section === 'h5p-hub-upload'}
-                onUpload={this.handleUpload} />
+              {
+                this.props.enableContentHub && 
+                  <UploadContent id="h5p-hub-upload"
+                    title={Dictionary.get('uploadTabLabel')}
+                    getAjaxUrl={this.props.getAjaxUrl}
+                    contentId={this.props.contentId}
+                    setFocus={this.state.expanded && this.state.section === 'h5p-hub-upload'}
+                    onUpload={this.handleUpload}
+                  />
+              }
```

Comment:
```suggestion
              <UploadContent id="h5p-hub-upload"
                title={Dictionary.get('uploadTabLabel')}
                getAjaxUrl={this.props.getAjaxUrl}
                contentId={this.props.contentId}
                setFocus={this.state.expanded && this.state.section === 'h5p-hub-upload'}
                onUpload={this.handleUpload}
              />
```

Reply:
The `UploadContent` component is always shown (it is for uploading .h5p files for your own use).